assets/scss/tabs.scss

```diff
@@ -0,0 +1,85 @@
+.nhsuk-bordered-tabs-container {
+  padding-top      : nhsuk-spacing(2);
+  background-color : $color_nhsuk-white;
+  border-bottom    : $nhsuk-border-width-form-element solid $color_nhsuk-grey-3;
+
+
+  .nhsuk-bordered-tabs {
+	padding       : 0;
+	margin-bottom : 0;
+	display       : flex;
+
+
+	.nhsuk-bordered-tabs-item {
+	  display       : inline-block;
+	  padding       : nhsuk-spacing(3) nhsuk-spacing(4);
+	  margin-bottom : 0;
+	  font-size     : 90%;
```

Comment:
How's that 90|% being judged? Looks like an unnecessary customisation/deviation to NHS frontend framework standards, which is probably not the best idea? Right?